src/routes/Courses/components/CoursesList.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import { Link, browserHistory } from 'react-router';
+
+class CoursesList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      courses: [
+        {description: "Firs course about cooking rissotto",
+        main_photo:  "1",
+        duration:  "34:54",
+        price:  "100$",
+        lessons: [{
+          video_link:  "1",
+          description: "1",
+          test: "ltest_id",
+        }
+      ]
+    },
+    {description: "Course make up like Shurigina",
+    main_photo:  "2",
+    duration:  "24:40",
+    price:  "20$",
+    lessons: [{
+      video_link:  "2.1",
+      description: "2.1",
+      test: "ltest_id2.1",
+    },
+    {
+      video_link:  "2.2",
+      description: "2.2",
+      test: "ltest_id2.2",
+    }
+  ]
+},
+]
+}
+};
+render() {
+
+  const coursesList = this.state.courses.map((item, i) =>
+
+<li  key={i}>
+    <div  className="col-xs-12 col-md-12">
+    <div className="col-xs-12 col-md-8">
+
+        <div className="col-xs-10">
+          <label  className="control-label col-xs-2">Description:</label>
+          <div> {item.description}</div>
+        </div>
+        <div className="col-xs-10">
+          <label  className="control-label col-xs-2">Main photo:</label>
+          <div> {item.main_photo}</div>
+        </div>  <div className="col-xs-10">
+          <label  className="control-label col-xs-2">Duration:</label>
+          <div> {item.duration}</div>
+        </div>  <div className="col-xs-10">
+          <label  className="control-label col-xs-2">Price:</label>
+          <div> {item.price}</div>
+        </div>
+
+    </div>
+    <div className="col-xs-12 col-md-4">
+      <button
+        type="button"
+        className="btn btn-primary lg"
+        onClick={() => { browserHistory.push(`/admin/courses/edit`)
+        }}
+        >Edit course</button>
+      </div>
+    </div>
+      </li>
+  );
+  return (
+    <div className="container">
+      <div className="row">
+        <div className="col-xs-6 col-md-12"  style={{ padding: 15}}>
+          <ul className="list-unstyled">
+            {coursesList}
+          </ul>
+          <button
+            type="button"
+            className="btn btn-success lg"
+            onClick={() => { browserHistory.push(`/admin/courses/new`)
+            }}
+            >New course</button>
```

Comment:
it looks better than link in top, so I'd like to keep button but move it above courses